packages/app-store/zoomvideo/api/add.ts

```diff
@@ -26,7 +26,7 @@ async function handler(req: NextApiRequest) {
   const params = {
     response_type: "code",
     client_id,
-    redirect_uri: `${WEBAPP_URL}/api/integrations/zoomvideo/callback`,
+    redirect_uri: `${WEBAPP_URL_FOR_OAUTH}/api/integrations/zoomvideo/callback`,
```

Comment:
Allows to install Zoom locally as WEBAPP_URL_FOR_OAUTH is localhost:3000 vs WEBAPP_URL which is app.cal.local.

Google Calendar also uses the same approach.

Also, In production env, WEBAPP_URL_FOR_OAUTH is same as WEBAPP_URL